### PR TITLE
#5 tcp connector

### PR DESCRIPTION
**Pull request by WingZer0o**
#5

### CODE REVIEW

File: data/tcp-connector.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
+/**
+ * @title TCP Connector: Ping
+ * @difficulty intermediate
+ * @tags cli
+ * @run --allow-net <url>
+ * @resource {https://deno.land/api?s=Deno.listen} Deno.listen: API
+ * @resource {https://github.com/denoland/denobyexample/blob/main/data/tcp-listener.ts} Example TCP server that will accept connections
```
Comment (kt3k) [inline data/tcp-connector.ts:7]: Let's use the site relative path
```suggestion
 * @resource {/tcp-listener} Example TCP server that will accept connections
```

File: data/tcp-connector.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
+/**
+ * @title TCP Connector: Ping
+ * @difficulty intermediate
+ * @tags cli
+ * @run --allow-net <url>
+ * @resource {https://deno.land/api?s=Deno.listen} Deno.listen: API
```
Comment (kt3k) [inline data/tcp-connector.ts:6]: typo?
```suggestion
 * @resource {https://deno.land/api?s=Deno.connect} Deno.connect: API
```

**Comment by WingZer0o** [comment]
Fixed those suggestions up.

**Review by kt3k — APPROVED** [review]
Thanks for updating! LGTM

**Pull request merged**